int32 device_id
int32 task_id
int32 map_id
float32 progress
int32 status
string message

